### PR TITLE
Add a speculative gimballess lr101

### PR DESCRIPTION
**Pull request by lpgagnon**
Maybe?

needs https://github.com/KSP-RO/RealismOverhaul/pull/2795

It's not the prettiest; couldn't find a transform for the remaining pipes:
Screenshot 2023-05-14 at 21 13 45

### CODE REVIEW

**Comment by Capkirk123** [comment]
Removing the yaw gimbal and base mesh completely makes it look a little messy (although removing just the base mesh and leaving the yaw mesh doesn't look much better)

File: GameData/ROEngines/PartConfigs/LR101_BDB.cfg [hunk 1]
```diff
@@ -164,10 +164,45 @@ PART
 	}
 }
 
-@PART[ROE-LR101Inline|ROE-LR101]:AFTER[FerramAerospaceResearch]
+
+
++PART[ROE-LR101Inline]:FIRST
+{
+	// A LR101 without gimbal assembly.
+	@name = ROE-LR101-Fixed
```
Comment (Capkirk123) [inline GameData/ROEngines/PartConfigs/LR101_BDB.cfg:172]: Should probably be ROE-LR101Fixed to maintain naming consistency

**Comment by lpgagnon** [comment]
> Removing the yaw gimbal and base mesh completely makes it look a little messy
It's definitely not high quality. My other option was a squished wac-corporal

Will fix the name.

**Comment by lpgagnon** [comment]
Turns out b9ps also allows offsetting and stretching transforms. So how about these 2 versions, as b9ps variants?

Screenshot 2023-05-14 at 23 37 06

**Comment by lpgagnon** [comment]
Less ugly with a tighter base showing just a bit of piping?
Screenshot 2023-06-07 at 23 22 37

**Comment by Capkirk123** [comment]
Yeah, not half-bad

**Comment by lpgagnon** [comment]
+1 commit (at last): fixed the name, added variant with base (matching the last screenshot)

**Pull request merged**